testing/kuttl/e2e/exporter/10-assert.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: exporter-tls
+status:
+  instances:
+    - name: instance1
+      readyReplicas: 1
+      replicas: 1
+      updatedReplicas: 1
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    postgres-operator.crunchydata.com/cluster: exporter-tls
+    postgres-operator.crunchydata.com/pgbackrest-backup: replica-create
+status:
+  succeeded: 1
```

Comment:
🤔 I don't see any backups being used in later steps. What does the assertion mean here?

Reply:
Again, I have left these in as general "cluster is stable" checks